nym-vpn-desktop/src/pages/location/NodeLocation.tsx

```diff
@@ -47,11 +48,16 @@ function NodeLocation({ node }: { node: NodeHop }) {
 
   // request backend to refresh cache
   useEffect(() => {
-    invoke<Country[]>('get_node_countries')
+    invoke<Country[]>('get_countries', {
+      nodeType: node === 'entry' ? 'Entry' : 'Exit',
```

Comment:
So it's already typed (inferred here by typescript, concrete type is `NodeHop`)

For instance that would raise TS errors:
```
node === 'BadValue' ? 'Entry' : 'Exit',
```

Reply:
> it would be better to have the Exit type explicit as well? 

I'd say no, there is no real added value to type tauri "commands" arguments, since it's cross language, you would have to maintain corresponding TS types to tauri args anyway and so I prefer not to do that

Lets keepit simple, no need to add a custom layer on top of tauri `invoke` API

Reply:
I was referring that you would fail before calling `invoke`, by checking that node is either `entry` or `exit`, so that it doesn't go on the `Exit` path in case it's different then `entry`. But since I'm not very familiar with TS, if it already does that by some typing magic, then it's all good.